src/SignalMessage.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace NotificationChannels\Signal;
+
+use Illuminate\Support\Arr;
+
+class SignalMessage
+{
+    /**
+    * The phone number messages will be sent from.
+    * Must include prefix ("+") and country code.
+    *
+    * @var string
+    **/
+    public $username;
+
+    /**
+    * The message content.
+    *
+    * @var string
+    **/
+    public $message;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param  string $message
+     *
+     * @return static string
+     */
+    public static function create(string $message = '')
+    {
+        return new static(string $message);
+    }
+
+    /**
+     * Create a new message instance.
+     *
+     * @param  string  $message
+     */
+    public function __construct($message = '')
+    {
+        $this->message = $message;
+    }
+
+    /**
+     * Set the message.
+     *
+     * @param  string  $message
+     *
+     * @return $this
+     */
+    public function message($message)
```

Comment:
still missing type hints on all the other methods here. Also, indenting is wonky as well.